### PR TITLE
[Bug 1083914] Changed the date input made by Datepicker to YYY-MM-DD

### PR DESCRIPTION
**Pull request by anushbmx**
Fixes the wrong date formate inserted by datepicker to create announcement form.

### CODE REVIEW

File: kitsune/sumo/static/sumo/js/wiki.dashboard.js [hunk 1]
```diff
@@ -30,8 +30,8 @@
 
     if ($('body').is('.localization-dashboard')) {
       // Add's datepicker to the create announcement pop-up
-      addDatePicker('#id_show_after');
-      addDatePicker('#id_show_until');
+      $('#id_show_after').datepicker('option', 'dateFormat', 'yy-mm-dd');
```
Comment (rehandalal) [inline kitsune/sumo/static/sumo/js/wiki.dashboard.js]: i'm not sure why you deleted the `addDatePicker` function. you should have just modified that.

File: kitsune/sumo/static/sumo/js/wiki.dashboard.js [hunk 2]
```diff
@@ -48,12 +48,6 @@
     });
   });
 
-  function addDatePicker(inputId) {
-    $(inputId).datepicker({
```
Comment (rehandalal) [inline kitsune/sumo/static/sumo/js/wiki.dashboard.js]: you could just change this to:
```javascript
$(inputId).datepicker('option', 'dateFormat', 'yy-mm-dd');
```

**Comment by anushbmx** [comment]
@rehandalal Jquery add's datepicker if the input type is set to date. So now you gonna ask why didnt this happen before, it's because we haven't included jquery before I added this.

It works on my clone you can have a look.

**Comment by rehandalal** [comment]
that doesn't really answer my question about why you deleted that function. to me since you're always using the same parameters it should just go into a single reusable function `addDatePicker`.

**Comment by anushbmx** [comment]
@rehandalal I got it, you mean change this inside the reusable function ok. My bad i missed :D 
Changes coming right up.

**Comment by anushbmx** [comment]
@rehandalal I have updated it now.

**Comment by rehandalal** [comment]
lgtm. thanks!

**Pull request merged**